Code/Framework/AzCore/AzCore/Math/Internal/SimdMathVec3_neon.inl

```diff
@@ -378,7 +378,7 @@ namespace AZ
         {
             // NeonQuad::CmpAllGt compares all 4 elements.
             // This makes sure unused fourth element of arg1 > arg2.
-            return NeonQuad::CmpAllLt(
+            return NeonQuad::CmpAllGt(
```

Comment:
Good spot! 😅

Reply:
Actually @greerdv figured it out very fast while he was helping me out. Thanks @greerdv!